core/src/player/state.rs

```diff
@@ -47,6 +47,11 @@ pub fn install(session: &mut GameSession) {
     add_state_module!(session, midair);
     add_state_module!(session, walk);
     add_state_module!(session, dead);
+
+    session
+        .stages
+        .add_system_to_stage(CoreStage::PreUpdate, incapacitated::handle_player_state)
+        .add_system_to_stage(PlayerStateStage, slippery_seaweed::update);
```

Comment:
Tiniest little change: it'd be good to move the registration of the slippery seaweed system to it's own `install` function.

That should be the last thing!

```suggestion
        .add_system_to_stage(CoreStage::PreUpdate, incapacitated::handle_player_state);
```

Reply:
I tried to do this first actually, as it makes more sense, but I get this error:
`thread 'main' panicked at 'Stage with label `PlayerStateStage` ( 01GP72HD5B1S7VDNE9B2SAGWVX ) doesn't exist.'`
Looks like this install function runs before the PlayerStateStage is created, not sure how to get around this.

Reply:
It worked now :)
(not sure how often you check discord, but I've asked you something on the jumpy channel, as you're awnsering fast here, just letting you know)